.github/workflows/hugegraph-python-client.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         pip install -r ./hugegraph-python-client/requirements.txt
     - name: Prepare HugeGraph Server Environment
       run: |
-        docker run -d --name=graph -p 8080:8080 hugegraph/hugegraph
+        docker run -d --name=graph -p 8080:8080 -e AUTH=true -e PASSWORD=admin hugegraph/hugegraph:1.2.0
```

Comment:
no need to set `-e AUTH-true` now (the latest PR https://github.com/apache/incubator-hugegraph/pull/2427/files enhance it)

Note: the docker-1.2.0 image need update first